docs/guide/repository.md

```diff
@@ -1549,6 +1549,124 @@ class AcmeFooBundle extends Bundle
 
 Policies used internally in repository services are defined in  `EzPublishCoreBundle/Resources/config/policies.yml` .
 
+### Custom Limitation UI
+
+To provide support for editing custom policies in Platform UI you need to implement [`EzSystems\RepositoryForms\Limitation\LimitationFormMapperInterface`](https://github.com/ezsystems/repository-forms/blob/master/lib/Limitation/LimitationFormMapperInterface.php):
+
+```php
+<?php
+
+/**
+ * Interface for LimitationType form mappers.
+ *
+ * It maps a LimitationType's supported values to editing form.
+ */
+interface LimitationFormMapperInterface
+{
+    /**
+     * "Maps" Limitation form to current LimitationType, in order to display one or several fields
+     * representing limitation values supported by the LimitationType.
+     *
+     * Implementors MUST either:
+     * - Add a "limitationValues" form field
+     * - OR add field(s) that map to "limitationValues" property from $data.
+     *
+     * @param FormInterface $form Form for current Limitation.
+     * @param Limitation $data Underlying data for current Limitation form.
+     */
+    public function mapLimitationForm(FormInterface $form, Limitation $data);
+    /**
+     * Returns the Twig template to use to render the limitation form.
+     *
+     * @return string
+     */
+    public function getFormTemplate();
+    /**
+     * This method will be called when FormEvents::SUBMIT is called.
+     * It gives the opportunity to filter/manipulate limitation values.
+     *
+     * @param Limitation $limitation
+     */
+    public function filterLimitationValues(Limitation $limitation);
+}
+```
+
+Next, register the service in DIC with the `ez.limitation.formMapper` tag and set the `limitationType` attribute to the Limitation type's identifier:
```

Comment:
can we expand DIC as it is used for the first time in the text

Reply:
Just in case: Acronym `DIC` stands for "Dependency Injection Container" ;-)